src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -7432,15 +7434,15 @@ def test_aks_azure_service_mesh_with_pluginca(self, resource_group, resource_gro
 
         # enable azure service mesh with pluginca
         update_cmd = 'aks mesh enable --resource-group={resource_group} --name={name} ' \
-                     '--key-vault-id my-akv-id ' \
+                     '--key-vault-id  {akv_resource_id} ' \
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=79450589&view=results) to validate the change.

Reply:
test passed